pkg/console/subresource/util/util.go

```diff
@@ -106,3 +106,9 @@ func HTTPS(host string) string {
 	secured := fmt.Sprintf("%s%s", protocol, host)
 	return secured
 }
+
```

Comment:
I think you can keep this func in the `configmap_builder.go` file.  It isn't shared elsewhere.

Reply:
Also, should we give it a unit test?

Reply:
will add !